### PR TITLE
add cni deep dive part 1 to blog

### PR DESCRIPTION
**Pull request by xmulligan**

### CODE REVIEW

**Review by liquidat — CHANGES_REQUESTED** [review]
I'd add that this covers a video.

File: src/posts/2022-02-23-cni-deep-dive/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2022-02-23T17:00:00.000Z'
+title: 'Cilium Kubernetes CNI Provider Deep Dive: Part 1'
+ogImageUrl: 'https://heise.cloudimg.io/v7/_www-heise-de_/imgs/18/2/9/5/4/2/1/8/cilium-diagramm-da030a781be5edc2.png?org_if_sml=1&q=70&width=1220'
+ogSummary: 'Learn what Cilium is and how it leverages eBPF to provide load balancing, observability, and security'
```
Comment (xmulligan) [inline src/posts/2022-02-23-cni-deep-dive/index.md:5]:
```suggestion
ogSummary: 'This video covers what Cilium is and how it leverages eBPF to provide load balancing, observability, and security'
```

**Review by liquidat — APPROVED** [review]

**Comment by liquidat** [comment]
@xmulligan There are build failures, can you please check?

File: src/posts/2022-02-23-cni-deep-dive/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2022-02-23T17:00:00.000Z'
+title: 'Cilium Kubernetes CNI Provider Deep Dive: Part 1'
+ogImageUrl: 'https://heise.cloudimg.io/v7/_www-heise-de_/imgs/18/2/9/5/4/2/1/8/cilium-diagramm-da030a781be5edc2.png?org_if_sml=1&q=70&width=1220'
```
Comment (xmulligan) [inline src/posts/2022-02-23-cni-deep-dive/index.md:4]:
```suggestion
ogImageUrl: 'https://docs.cilium.io/en/stable/_images/cilium-arch.png'
```

**Comment by xmulligan** [comment]
@liquidat should be good now

File: src/posts/2022-02-23-cni-deep-dive/index.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2022-02-23T17:00:00.000Z'
+title: 'Cilium Kubernetes CNI Provider Deep Dive: Part 1'
+ogImageUrl: 'https://docs.cilium.io/en/stable/_images/cilium-arch.png'
+ogSummary: 'This video covers what Cilium is and how it leverages eBPF to provide load balancing, observability, and security'
```
Comment (xmulligan) [inline src/posts/2022-02-23-cni-deep-dive/index.md:5]:
```suggestion
ogSummary: 'This video from The Learning Channel covers what Cilium is and how it leverages eBPF to provide load balancing, observability, and security'
```

**Pull request merged**